study.md

```diff
@@ -22,7 +22,9 @@ material. Instead, digest what you've read and repeat it in your own voice.
 In your own words, define ORM and explain why using an ORM is valuable.
 
 ```md
-<!-- your response here -->
+ORM is object relational mapping. It connects the objects in application tables to the databases.
```

Comment:
Object relational-mapping (ORM) is a method of representing arbitrary data as objects in a programming language.